experimental/sql/test/projection_pushdown_tests/order_by_extended.xdsl

```diff
@@ -0,0 +1,32 @@
+// RUN: rel_opt.py -p projection-pushdown %s | filecheck %s
+
+module() {
+  rel_alg.aggregate() ["col_names" = ["a"], "functions" = ["sum"], "res_names" = ["a_sum"], "by" = []] {
+    rel_alg.order_by() ["by" = [!rel_alg.order<"a", "desc">, !rel_alg.order<"b", "asc">]] {
```

Comment:
Completely off-topic but there is an opportunity to drop order-bys that don't have an effect, such as this one. 

With aggregate, you have to know that the aggregate function is commutative, which, if you consider overflows, might not actually be the case. But there might be other cases, such as two subsequent order-bys, or an order-by followed by a join (if we assume that the join produces its own order and isn't required to preserve the existing one.)

Never mind, just my brain producing ideas... 😉